File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
         height = get_int ("Height: \n");
    }
    while (height<1 || height>8);

    for (int j = 0; j < height; j++)
    {
        for (int k = height - 1; k > j ; k--)
        {
            printf (" ");
        }

        for (int i=-1; i < j ; i++)
        {
            printf ("#");
        }

        printf ("  ");

         for (int l=-1; l < j ; l++)
        {
            printf ("#");
        }

        printf ("\n");
    }
}

File: sentimental-mario-less/mario.py
from cs50 import get_int
from sys import exit

while True:
    n = get_int("Height: ")
    if n > 1 or n < 9:
        exit(0)


k = n
for i in range(n):
    k -= 1
    print(" " * k + "#" * (i + 1))

